libs/blocks/global-navigation/global-navigation.js

```diff
@@ -519,7 +519,7 @@ class Gnav {
       return 'linux';
     };
 
-    const unavVersion = new URLSearchParams(window.location.search).get('unavVersion') || '1.1';
+    const unavVersion = new URLSearchParams(window.location.search).get('unavVersion') || config.unavVersion || '1.1';
```

Comment:
What would be the reason for allowing this to be configurable?

Reply:
This was a request from the client in case they needed time to verify their own navigation before upgrading the Unav version. Since we don't have versioning in our standalone code, we provide whatever is in our main branch, which means they don't have an option to opt out of the upgrade.

Reply:
Does that consumer have any special overwrites on top of the base nav? E.g. changes styles that aren't offered by you?

Ideally testing should happen centrally (by tattva, verifying that all consumers work) - and there shouldn't be any cause for concern that they need to do validations themselves, unless they really hacked into the gnav OR the unav has frequent breaking changes, which otherwise can't be rolled out.

If the unav does have a lot of breaking changes, that's a discussion to be had with them to discourage them from that or provide a way of opting into the breaking changes.

Reply:
I don't see a lot of contexts of the request, but I'd further enquire the consumer about his reasoning and trying to make central improvements, such as adding that consumer to a list of consumers to take extra care on for testing.

I'm not blocking the PR in any way, I would just further question on why it's needed, before opening this box-of-pandora - if you expect unav v1.5 for example, and consumers are on v1.3 you might have compatibility issues in the future on your end, when shipping new features